samples/unit-tests/axis/breaks/demo.js

```diff
@@ -704,3 +704,663 @@ QUnit.test('Axis breaks on Y axis', function (assert) {
         '50 and 100 translate to the same axis position'
     );
 });
+
+QUnit.test('#14236: Stacked area chart null yBottom', assert => {
+    const chart = Highcharts.chart('container', {
+        xAxis: {
+            type: 'datetime'
+        },
+        plotOptions: {
+            area: {
+                stacking: 'normal',
+                connectNulls: true
+            },
+            series: {
+                gapSize: 1
+            }
+        },
+        series: [
+            {
+                type: 'area',
+                data: [
```

Comment:
That's a lot of data... Test samples should be small for two reasons: easier to read and understand for future us, and small data performs better so we reduce the time required to run the tests. This setup is sufficient to reproduce the bug:

```js
const chart = Highcharts.chart('container', {
    plotOptions: {
        area: {
            stacking: 'normal',
            connectNulls: true
        },
        series: {
            gapSize: 1
        }
    },
    series: [{
            type: 'area',
            data: [
                [0, 0],
                [1, 0],
                [3, 0],
                [4, 0]
            ]
        },
        {
            type: 'area',
            data: [
                [0, 1],
                [1, 1],
                [2, 1],
                [3, 1],
                [4, 1]
            ]
        }
    ]
});
```